service/src/main/java/bio/terra/workspace/service/resource/controlled/flight/clone/bucket/CreateStorageTransferServiceJobStep.java

```diff
@@ -114,29 +122,28 @@ private void createTransferJob(
       BucketCloneInputs sourceInputs,
       BucketCloneInputs destinationInputs,
       String transferJobName,
-      String controlPlaneProjectId,
-      Storagetransfer storageTransferService)
+      String controlPlaneProjectId)
       throws IOException {
     final TransferJob transferJobInput =
         new TransferJob()
             .setName(transferJobName)
-            .setDescription("Terra Workspace Manager Clone GCS Bucket")
+            .setDescription(TRANSFER_JOB_DESCRIPTION)
             .setProjectId(controlPlaneProjectId)
             .setSchedule(createScheduleRunOnceNow())
             .setTransferSpec(
                 createTransferSpec(sourceInputs.getBucketName(), destinationInputs.getBucketName()))
             .setStatus(ENABLED_STATUS);
     // Create the TransferJob for the associated schedule and spec in the correct project.
     final TransferJob transferJobOutput =
-        storageTransferService.transferJobs().create(transferJobInput).execute();
+        storagetransfer.transferJobs().create(transferJobInput).execute();
     logger.debug("Created transfer job {}", transferJobOutput);
   }
 
   // Remove the Storage Transfer Job. Any data in the destination bucket will be deleted in the
   // previous step's undo method.
   @Override
   public StepResult undoStep(FlightContext flightContext) throws InterruptedException {
-    return StorageTransferServiceUtils.deleteTransferJobStepImpl(flightContext);
+    return StorageTransferServiceUtils.deleteTransferJobStepImpl(flightContext, storagetransfer);
```

Comment:
A failure here will be a DISMAL FAILURE from Stairway. Maybe that is the right thing; perhaps add a comment to that effect?

Reply:
I think I was wrong, though. The utility method does not error, it logs and continues. Thought I deleted this comment.